lib/Apply_Trace.thy

```diff
@@ -8,8 +8,7 @@
  * Provides an alternate refinement function which takes an additional stateful journaling operation. *)
 theory Apply_Trace
 imports
-  Main
-  "MLUtils"
+  ML_Utils.MLUtils
```

Comment:
Do we not want these to be named consistently? `ML_Utils` vs `MLUtils` on the same line is a bit blah

Reply:
Other than that, looks good to me.

Reply:
Didn't want to introduce too much noise, but you do have a point. I'll add that rename.